packages/firebase_crashlytics/firebase_crashlytics/ios/Classes/FLTFirebaseCrashlyticsPlugin.m

```diff
@@ -68,6 +69,9 @@ + (instancetype)sharedInstance {
     instance = [[FLTFirebaseCrashlyticsPlugin alloc] init];
     // Register with the Flutter Firebase plugin registry.
     [[FLTFirebasePluginRegistry sharedInstance] registerFirebasePlugin:instance];
+    [[FIRCrashlytics crashlytics] setDevelopmentPlatformName:@"Flutter"];
+    // We can't currently get the Flutter plugin version number, so use -1.
+    [[FIRCrashlytics crashlytics] setDevelopmentPlatformVersion:@"-1"];
```

Comment:
We can inject this version if necessary through the podspec `GCC_PREPROCESSOR_DEFINITIONS`.

I remember seeing something like shell/version/version.h in Flutter engine that defined this as well, but doesn't seem like it's available here 🙈 

Happy to ship as is though - but I can also push something for the podspec injection of the version if necessary, let me know. Thanks

Reply:
It would definitely be helpful to have the version number, though less necessary for this initial update. If you have time to push something with the version, that would be great but also happy to wait on that if needed.

Reply:
I'll take a look at this

Reply:
I'm going to to get this shipped today, have added a task in our internal backlog to pickup injecting the version (will get picked up after the release next week)